luminoso_api/v5_download.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import csv
+from signal import signal, SIGPIPE, SIG_DFL
+from urllib.parse import urlparse
+from tqdm import tqdm as progress
+
+from .v5_client import LuminosoClient
+from .v5_constants import URL_BASE
+
+# Python raises IOError when reading process (such as `head`) closes a pipe.
+# Setting SIG_DFL as the SIGPIPE handler prevents this program from crashing.
+signal(SIGPIPE, SIG_DFL)
+
+
+DESCRIPTION = "Download documents from a Luminoso project via the command line."
+DOCS_PER_BATCH = 1000
```

Comment:
We can get up to 25000 documents at a time with the API.  Was this number determined to be faster than getting more at a time and making fewer calls?

Reply:
As long as I'm paginating, I figured that smaller pages were nicer to the API, and also this allows progress to update more often.